src/mixins/common-mixin.ts

```diff
@@ -68,4 +74,27 @@ export default class CommonMixin extends Vue {
     }
     return true
   }
+
+  /**
+   * Navigates to the dashboard page, optionally with a filing ID.
+   * @param identifier The identifier to include in the dashboard URL
+   * @param filingId The filing ID to include in the dashboard URL (optional, defaults to null)
+   */
+  protected navigateToDashboard (identifier: string, filingId: number | null = null): void {
```

Comment:
Can we give this function a more specific name? Might be a bit generic here. I'm thinking, `navigateToNewDashboard` or `navigateToBusinessDashboard`? What do you think?

Reply:
Thanks, agree. I will update it to navigateToBusinessDashboard

Reply:
Thanks! Resolving.